src/image_transforms.rs

```diff
@@ -1,7 +1,28 @@
-use image::imageops::{Nearest, Triangle};
+use image::imageops::Triangle;
 use image::io::Reader as ImageReader;
 use image::{DynamicImage, GenericImageView, ImageFormat};
-use std::{fs, os::unix::fs::MetadataExt};
+use std::{fs, io::BufReader, os::unix::fs::MetadataExt};
+
+use exif;
+
+pub fn read_exif_metadata(image_path: &str) -> Option<u32> {
+    let file = fs::File::open(image_path).unwrap();
+
+    let exif_metadata = exif::Reader::new().read_from_container(&mut BufReader::new(&file));
+
+    // If read_from_container results in Err, this means that there is no EXIF metadata
+    // https://github.com/kamadak/exif-rs/issues/24
+    match exif_metadata {
```

Comment:
Since you return None in the error case, would `if let Some(...)` make it easier?